### ISSUE TITLE
[BUG/Help] ptuning过程中loss陡降至0

### ISSUE THREAD

**Issue by yucc-leon** [THUDM/ChatGLM-6B#1327]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![image](https://github.com/THUDM/ChatGLM-6B/assets/17897916/d25d0676-cf2c-44c7-b249-4ffba4c8b491)
如图，使用私有数据训练，不是100%必现，在某些数据组合中会出现中从2.x突然降到0.0的情况。
读取checkpoint测试，输出都是<UNK>。已知跟learning rate设置、数据集都有关系
![image](https://github.com/THUDM/ChatGLM-6B/assets/17897916/15777c3a-47c0-4d6b-a342-590160d756c7)

主要好奇loss为0在这里代表了什么含义。

### Steps To Reproduce

learning rate = 2e-2
私有数据为考试题目，暂时不便提供。

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by zzy347964399** [comment]
（1）打印原来数据的input_ids和labels看看是不是有nan，infinite，指令为
`# print(torch.isnan(tesnor_input))

print(torch.isfinite(tesnor_input).int().sum())`

（2）精度相关
（3）梯度爆炸了


另外就是，？？可以用model.half()来解决
